docs/source/benchmark.rst

```diff
@@ -89,8 +89,9 @@ the iterative learner of `ndl2`. ``R -f run_ndl.R`` runs the `ndl2` on the same
 while `ndl` only runs on the smaller files because it is orders of magnitude slower.
 
 
-`ndl` can be installed from CRAN by running ``install.packages('ndl')`` in R, while `ndl2` is just
-available upon request (contact `Konstantin Sering <https://uni-tuebingen.de/fakultaeten/philosophische-fakultaet/fachbereiche/neuphilologie/seminar-fuer-sprachwissenschaft/arbeitsbereiche/quantitative-linguistik/mitarbeiter/'>`_, install with ``R CMD INSTALL ndl2.tar.gz``).
+`ndl` can be installed from CRAN by running ``install.packages('ndl')`` in R,
+while `ndl2` needs to be downloaded from Github first (`ndl2
+<https://github.com/quantling/ndl2>`_, install with ``R CMD INSTALL ndl2.tar.gz``).
```

Comment:
Isn't there an option to install R packages from Github directly?

Reply:
Yes, in principle it is possible to run ``github_install('quantling/ndl2')`` but this only succeed, if all dependencies are installed (as far as I understand). If you download the package from github manually you are exposed to the instructions that dependencies exist.

Reply:
I see. I just remembered that ''back in the days'', I used `devtools::install_github()` a few times and it seemed to work, but I do not remember anymore whether you had to install the dependencies manually or not. Probably, its worth trying out.